pyproject.toml

```diff
@@ -23,11 +23,12 @@ maintainers = [
 
 dependencies = [
     "arviz_plots @ git+https://github.com/arviz-devs/arviz-plots",
-    "formulae>=0.5.3",
+    "formulae @ git+https://github.com/bambinos/formulae",
```

Comment:
Needs to be updated before making a release (for which we need a formulae release)